lib/clearance/engine.rb

```diff
@@ -1,12 +1,19 @@
-require 'clearance'
-require 'rails'
+require "clearance"
+require "rails"
 
 module Clearance
   class Engine < Rails::Engine
-    initializer 'clearance.filter' do |app|
+    initializer "clearance.filter" do |app|
       app.config.filter_parameters += [:password, :token]
     end
 
-    config.app_middleware.insert_after ActionDispatch::ParamsParser, Clearance::RackSession
+    config.app_middleware.insert_after(
+      ActionDispatch::ParamsParser,
+      Clearance::RackSession
```

Comment:
I wasn't even aware you could have trailing commas in method calls. I tried it out and it does work, but I don't think I've seen that convention outside of arrays and hashes.